### DISCUSSION TITLE
Article: Exploring Angular CDK: Creating Context Menu & Text Popover | Angular Material Dev

### DISCUSSION THREAD

**Discussion by giscus** [Angular-Material-Dev/community#9] [Article and Course Discussions]
# Article: Exploring Angular CDK: Creating Context Menu & Text Popover | Angular Material Dev

Learn Material Design in Angular, Theming Angular Material Components and Developing your Angular application with Material Design like a pro.

https://angular-material.dev/articles/exploring-cdk-context-menu-text-popover

**Comment by abelfubu** [comment]
Good article Dharmen! Those new CDK features are dope!

**Reply by shhdharmen** [reply]
I am glad that you liked it!

If you want to purchase the code, I am giving 25% introductory discount. Use discount code **A3MDM0NA** while checking out from [my storefront](https://angular-material-dev.lemonsqueezy.com/).

Note that discount is valid only for 1st 50 customers!
